src/internal_events/reduce.rs

```diff
@@ -9,3 +10,26 @@ impl InternalEvent for ReduceStaleEventFlushed {
         counter!("stale_events_flushed_total").increment(1);
     }
 }
+
+#[derive(Debug)]
+pub struct ReduceAddEventError {
+    pub error: PathParseError,
+}
+
+impl InternalEvent for ReduceAddEventError {
+    fn emit(self) {
+        error!(
+            message = "Event could not be reduced.",
+            error = ?self.error,
+            error_type = error_type::CONDITION_FAILED,
+            stage = error_stage::PROCESSING,
```

Comment:
```suggestion
            error = ?self.error,
            error_type = error_type::CONDITION_FAILED,
            stage = error_stage::PROCESSING,
            path = &self.path
```

Could we add the invalid `path` here too? I think that could be useful for debugging.